google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingConfig.java

```diff
@@ -134,7 +147,7 @@ private Formatter getFormatterProperty(String name, Formatter defaultValue) {
     String stringFilter = getProperty(name);
     try {
       if (stringFilter != null) {
-        Class clz = ClassLoader.getSystemClassLoader().loadClass(stringFilter);
+        Class<?> clz = ClassLoader.getSystemClassLoader().loadClass(stringFilter);
```

Comment:
nit: Is there any change to `loadClass` function in this PR? I just wonder why this change is here...

Reply:
This change eliminates a warning. Nothing else.